### PR TITLE
feat(wait-Light): adjust msleep to 30ms

### PR DESCRIPTION
**Pull request by aarxnxx**
adjust msleep to 30ms

### CODE REVIEW

File: src/lego_robot/main.c [hunk 1]
```diff
@@ -68,7 +68,7 @@ void follow_black_line(int motor_a, int motor_b)
 void drive_to_start()
 {
     enable_servo(0);
-    msleep(10);
+    msleep(30);
```
Comment (schmaenjael) [inline src/lego_robot/main.c:71]: Why is it important to increase the delay?

File: src/lego_robot/main.c [hunk 2]
```diff
@@ -68,7 +68,7 @@ void follow_black_line(int motor_a, int motor_b)
 void drive_to_start()
```
Comment (schmaenjael) [inline src/lego_robot/main.c:68]: Rename this function to sound cooler!

**Review by schmaenjael — CHANGES_REQUESTED** [review]
Unnecessary changes!

File: src/lego_robot/main.c (hunk 1 above)
Comment (aarxnxx) [reply src/lego_robot/main.c:71]: Fortnite

File: src/lego_robot/main.c [hunk 3]
```diff
@@ -65,10 +65,10 @@ void follow_black_line(int motor_a, int motor_b)
         drive(0, motor_b, 10);
 }
 
-void drive_to_start()
+void cool_drive_to_start()
```
Comment (schmaenjael) [inline src/lego_robot/main.c:68]: PerfecT!

**Review by schmaenjael — APPROVED** [review]
LGTM!

**Pull request closed without merging**